content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -169,26 +169,27 @@ supported path types:
-| Prefix | `/aaa/bb`                       | `/aaa/bbb`                    | No                                 |
-| Prefix | `/aaa/bbb`                      | `/aaa/bbb`                    | Yes                                |
-| Prefix | `/aaa/bbb/`                     | `/aaa/bbb`                    | Yes, ignores trailing slash        |
-| Prefix | `/aaa/bbb`                      | `/aaa/bbb/`                   | Yes,  matches trailing slash       |
-| Prefix | `/aaa/bbb`                      | `/aaa/bbb/ccc`                | Yes, matches subpath               |
-| Prefix | `/aaa/bbb`                      | `/aaa/bbbxyz`                 | No, does not match string prefix   |
-| Prefix | `/`, `/aaa`                     | `/aaa/ccc`                    | Yes, matches `/aaa` prefix         |
-| Prefix | `/`, `/aaa`, `/aaa/bbb`         | `/aaa/bbb`                    | Yes, matches `/aaa/bbb` prefix     |
-| Prefix | `/`, `/aaa`, `/aaa/bbb`         | `/ccc`                        | Yes, matches `/` prefix            |
-| Prefix | `/aaa`                          | `/ccc`                        | No, uses default backend           |
-| Mixed  | `/foo` (Prefix), `/foo` (Exact) | `/foo`                        | Yes, prefers Exact                 |
+| Kind   | Path(s)                         | Request path(s)               | Matches?                            |
+|--------|---------------------------------|-------------------------------|-------------------------------------|
+| Prefix | `/`                             | (all paths)                   | Yes                                 |
+| Exact  | `/foo`                          | `/foo`                        | Yes                                 |
+| Exact  | `/foo`                          | `/bar`                        | No                                  |
+| Exact  | `/foo`                          | `/foo/`                       | No                                  |
+| Exact  | `/foo/`                         | `/foo`                        | No                                  |
+| Prefix | `/foo`                          | `/foo`, `/foo/`               | Yes                                 |
+| Prefix | `/foo/`                         | `/foo`, `/foo/`               | Yes                                 |
+| Prefix | `/aaa/bb`                       | `/aaa/bbb`                    | No                                  |
+| Prefix | `/aaa/bbb`                      | `/aaa/bbb`                    | Yes                                 |
+| Prefix | `/aaa/bbb/`                     | `/aaa/bbb`                    | Yes, ignores trailing slash         |
+| Prefix | `/aaa/bbb`                      | `/aaa/bbb/`                   | Yes,  matches trailing slash        |
+| Prefix | `/aaa/bbb`                      | `/aaa/bbb/ccc`                | Yes, matches subpath                |
```

Comment:
Note: There is an open question about prefix `/aaa/bbb` matching paths:
`/aaa/bbb/ccc` and `/aaa/bbb/ccc/ddd`

Ping @kubernetes/sig-network-misc

Reply:
@kubernetes/sig-network-pr-reviews 👋🏻